esp/services/ws_access/ws_accessService.cpp

```diff
@@ -2075,7 +2075,7 @@ bool Cws_accessEx::onResourceDelete(IEspContext &context, IEspResourceDeleteRequ
             resp.setPrefix(req.getPrefix());
         }
         SecResourceType rtype = str2type(basednReq->getRtype());
-        Owned<IMultiException> me = MakeMultiException("onResourceDelete");
```

Comment:
@RussWhitehead do we still need this part of the changes?

Reply:
@wangkx We do not, as this was previously used to forward what used to be an exception up the caller stack